app/src/main/java/com/ciandt/dragonfly/example/DragonflyApplication.kt

```diff
@@ -56,5 +58,10 @@ class DragonflyApplication : Application() {
     private fun setupDragonflyLib() {
         DragonflyLogger.setLogLevel(DragonflyLogger.LOG_LEVEL_DEBUG)
         DragonflyConfig.shouldSaveBitmapsInDebugMode(Features.SAVE_CAPTURED_IMAGES_TO_DEVICE)
+
+        val dropboxPath = Environment.getExternalStorageDirectory().getAbsolutePath() + File.separator + BuildConfig.APPLICATION_ID;
```

Comment:
The place where the files will be "dropped" to be processed. I'm  open to suggestions :)